src/connectors/atomService.ts

```diff
@@ -227,8 +235,24 @@ export class AtomService extends DataSource {
    *
    * A Prisma like method for updating many records.
    */
-  updateMany = async ({ table, where, data, columns = '*' }: UpdateInput) =>
-    this.knex.where(where).update(data).into(table).returning(columns)
+  updateMany = async ({
+    table,
+    where,
+    whereIn,
+    data,
+    columns = '*',
+  }: UpdateManyInput) => {
+    const action = this.knex
+
+    if (where) {
+      action.where(where)
+    }
+    if (whereIn) {
+      action.whereIn(...whereIn)
+    }
```

Comment:
if no `where` nor `whereIn` this will become dangerous of changing all rows:

    UPDATE table SET data ...